metron-interface/metron-alerts/e2e/login/login.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LoginPage } from './login.po';
+
+describe('login to application', function() {
+    let page: LoginPage;
+
+    beforeEach(() => {
+        page = new LoginPage();
+    });
+
+    it('should display error message for invalid credentials', () => {
```

Comment:
Yes, I need to connect to a rest-api source for authentication. I hard coded it to node1:8082 since quick-dev and full-dev create's boxes with this name. I can make it configurable if you see an issue with this. I thought it would just make things simple.